src/App.ts

```diff
@@ -0,0 +1,118 @@
+import * as net from 'net';
+const fs = require('fs');
+const electron = require('electron');
+const path = require('path');
+const url = require('url');
+
+
+export class App
+{
+
+    public listen()
+    {
+        let num: number = 0;
+        let path: string = './Downloads';
+        var exist = this.statPath(path);
+        if(!exist)
+        {
```

Comment:
One line `if` and `for` statements can be used with brackets. You can remove them